tui-scrollview/src/state.rs

```diff
@@ -82,4 +82,14 @@ impl ScrollViewState {
             .map_or(u16::MAX, |size| size.height.saturating_sub(1));
         self.offset.y = bottom;
     }
+
+    /// True if the scroll view state is at the bottom of the buffer
+    /// Takes the pagesize into account
```

Comment:
nit: Wrap at 100 chars. First line should be a summary sentence followed by a newline then any detailed info.

Here, I think it's worth adding some info on the caveats such as if the size / page size is not yet computed (by rendering), then it won't be taken into account. Likely this info should be also added to scroll_to_bottom docs too.

Reply:
I can't wrap at 100 chars as the line is not longer than 100 chars. Or my editor lies to me :shaking_face: 

I added some info on the behavior of both functions. While doing so, I figured it would be best to just define a known behavior in that case. Changing the default for the buffer from `u16:Max` to `0` ensures that the function always returns true in that case. Also spotted a potential panic/undefined behavior, if the scroll position is at `u16:Max` and pagesize is `>= 1`, the computation would overflow the underlying `u16`. Saturated add seems fine to me, objections?

Reply:
> As it stands wrapping with a single line would not cause a new paragraph, so the second line is a continuation of the first. It's wrapped too soon.


Is it correct now?